sample-kotlin/build.gradle

```diff
@@ -35,7 +35,9 @@ dependencies {
     compile (project(':library')) {
         exclude module: "support-v13"
     }
-    compile project(':processor')
+
+    provided project(':processor')
 
     kapt project(':processor')
```

Comment:
Since I didn't read your code using IDE, I may be incorrect, but `PermissionRequest` is in library module.